charts/prometheus-blackbox-exporter/values.yaml

```diff
@@ -46,6 +46,9 @@ tolerations: []
 affinity: {}
 
 secretConfig: false
+# if the configuration is managed as secret outside the chart, using SealedSecret for example,
+# provide the name of the secret here
+configExistingSecretName: ""
```

Comment:
Maybe state that this is ignored if secretConfig is set ? (and document what is secretConfig, as those two could be misunderstood)

Reply:
I  changed the comment and the order of variable to group secretConfig and config as they work together, is it better now ?

Reply:
Did you push? I don't see the changes.

Reply:
my bad, forgot to push...